doc/CloudIntegration.xml

```diff
@@ -0,0 +1,382 @@
+                <para>Manufacturer Cloud Service</para>
+              </entry>
+            </row>
+            <row>
+              <entry valign="middle">
+                <para>OCS</para>
+              </entry>
+              <entry valign="middle">
+                <para>Operational Cloud Service</para>
+              </entry>
+            </row>
+            <row>
+              <entry valign="middle">
+                <para>PCS</para>
+              </entry>
+              <entry valign="middle">
+                <para>Provisioning Cloud Service</para>
+              </entry>
+            </row>
+            <row>
+              <entry valign="middle">
+                <para>VMS</para>
+              </entry>
+              <entry valign="middle">
+                <para>Video Management System</para>
+              </entry>
+            </row>
+          </tbody>
+        </tgroup>
+      </informaltable>
+    </section>
+  </chapter>
+  <chapter>
+    <title>Overview</title>
+    <para>On-boarding devices in cloud native Video Management Systems involves two major steps:</para>
+    <itemizedlist>
+      <listitem>
+        <para>A device taken out of the box connects with its pre-programmed parameters to a Manufacturer Cloud Service (MCS), so that the user can take ownership of the device. Once the device is successfully claimed by the user, the MCS behaves as a Provisioning Cloud Service (PCS).</para>
+      </listitem>
+      <listitem>
+        <para>The user can transfer the device from a PCS to an Operational Cloud Service (OCS), i.e. the cloud native VMS the user expects to use for his surveillance system.</para>
+      </listitem>
+    </itemizedlist>
+    <para>N.B. that at any time, a OCS may act as a PCS.</para>
+    <para>The scope of this cloud service covers only the second step, i.e. device sharing or transferring between different cloud services. The procedure outlined in these specifications can be used to share or transfer a device from a MCS to an OCS during the initial setup, but also from an old PCS to a new OCS.</para>
+    <para>Figure <xref xmlns:xlink="http://www.w3.org/1999/xlink" linkend="_Ref752965479"/> demonstrates the two phases. N.B. the gray rectangle visually identifies the scope of these specifications.</para>
+    <figure xml:id="_Ref752965479">
+      <title>Onboarding phases and ONVIF scope</title>
+      <mediaobject>
+        <imageobject>
+          <imagedata fileref="media/CloudIntegration/onboarding_scope.svg" contentwidth="159.97mm" />
+        </imageobject>
+      </mediaobject>
+    </figure>
+    <para>If the original connection to the MCS is kept, it is possibile for the MCS to perform FW
+      upgrade, upon request from the OCS. On the MCS, mutliple versions of FW images may be
+      present for each model, so that the OCS can select the desired one. In case upgrading to the
+      desired FW version will require multiple operations, the device and the MCS will performed
```

Comment:
```suggestion
      desired FW version will require multiple operations, the the MCS will manage the device upgrade  
```

Reply:
I changed from performed to perform. See if it sounds better.